environment.yml

```diff
@@ -4,6 +4,7 @@ channels:
   - bioconda
   - defaults
 dependencies:
+  - python <3.12,>=3.9
```

Comment:
A bit picky, but for the sake of consistency, could you please switch python version so they appear in the right order?  -> `python >=3.9, <3.12`

Reply:
sure thing!